testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/AppInitiatedActionUpdateProfileTest.java

```diff
@@ -111,6 +113,8 @@ public void updateProfileLoginFirst() {
         updateProfilePage.update("New first", "New last", "new@email.com", "test-user@localhost");
 
         events.expectLogin().assertEvent();
+        events.expectRequiredAction(EventType.UPDATE_PROFILE).detail(Details.PREVIOUS_FIRST_NAME, "Tom").detail(Details.UPDATED_FIRST_NAME, "New first").assertEvent();
+        events.expectRequiredAction(EventType.UPDATE_PROFILE).detail(Details.PREVIOUS_LAST_NAME, "Brady").detail(Details.UPDATED_LAST_NAME, "New last").assertEvent();
         events.expectRequiredAction(EventType.UPDATE_EMAIL).detail(Details.PREVIOUS_EMAIL, "test-user@localhost").detail(Details.UPDATED_EMAIL, "new@email.com").assertEvent();
         events.expectRequiredAction(EventType.UPDATE_PROFILE).assertEvent();
```

Comment:
The PR looks better now, however it still triggers 3 UPDATE_PROFILE events for each update of user, which is not ideal IMO. It will be better to have just single UPDATE_PROFILE event for each update, but just have the fields for firstName and lastname as the details. So for example for this test method "updateProfileLoginFirst()" , you won't have 2 lines 116 and 117 with 2 UPDATE_PROFILE events, but you will have just the single UPDATE_PROFILE event with the details similar to this:
```
events.expectRequiredAction(EventType.UPDATE_PROFILE)
    .detail(Details.PREVIOUS_FIRST_NAME, "Tom").detail(Details.UPDATED_FIRST_NAME, "New first")
    .detail(Details.PREVIOUS_LAST_NAME, "Brady").detail(Details.UPDATED_LAST_NAME, "New last")
    .assertEvent();
```

Is it possible to do it this way in this PR?